client/pom.xml

```diff
@@ -42,6 +42,35 @@
             <artifactId>javax.inject</artifactId>
             <version>${javax.inject.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.google.inject</groupId>
+            <artifactId>guice</artifactId>
+            <version>${guice.version}</version>
+        </dependency>
+        <dependency>
```

Comment:
Since the client jar will actually be imported by the user, I wanted to keep the dependencies with the parent pom to an absolute minimum.
I was thinking of actually using the actual version number rather than the variable :)